### ISSUE TITLE
[Fixed in GH-6052] Missing Visual Focus on My Account: Order History

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4329]
![Screen Shot 2019-08-29 at 8 41 06 PM](https://user-images.githubusercontent.com/15113219/63985372-6c782480-ca9d-11e9-9ec9-786c5305a2b7.png)

**Comment by Zeyber** [comment]
Pagination section is now tabable

**Comment by KateChuen** [comment]
On Chrome:
There's the visual focus everywhere where need be. But the visual focus on pagination is red, is that as per design? 
Also, cannot select the buttons of pagination. 

![Screen Shot 2019-10-21 at 3 05 39 PM](https://user-images.githubusercontent.com/45409515/67235296-d1545900-f414-11e9-82f3-68764dc12de2.png)




On Safari:
Same issue with the pagination as chrome. 
Also, missing visual focus and keyboard accessibility to the orders. 

On firefox:
Missing visual focus and keyboard accessibility on pagination AND orders. 
And the visual focus on the "Sort" menu is partly missing when it's expanded. 

<img width="1138" alt="Screen Shot 2019-10-21 at 3 10 03 PM" src="https://user-images.githubusercontent.com/45409515/67235345-e6c98300-f414-11e9-9f2f-a2faab4e18d3.png">

**Comment by developpeurweb** [comment]
This now works cross-browser:
![Screen Shot 2020-01-29 at 6 17 52 PM](https://user-images.githubusercontent.com/15113219/73406152-bcb11800-42c3-11ea-9a52-0de369b251c0.png)

Visible focus partly missing when expanding select menus is the expected behaviour since contents are selectable with the spacebar and arrow keys.

**Comment by giancorderoortiz** [comment]
I am unable to see visual focus on Firefox. v73.0.1
Needs work.

**Comment by developpeurweb** [comment]
This works on my end in Firefox and dev-3, although I'll profit on the review to punch-out the focus indicator as it seems partially hidden.

![Screen Shot 2020-02-18 at 3 55 09 PM](https://user-images.githubusercontent.com/15113219/74777239-12476780-5267-11ea-87be-81919a3a037a.png)

**Comment by developpeurweb** [comment]
This is now fixed for Firefox

**Comment by KateChuen** [comment]
Tested on dev10. 
Verified on chrome, safari & firefox. Focus on pagination works now. 
<img width="353" alt="Screen Shot 2020-03-27 at 10 04 51 AM" src="https://user-images.githubusercontent.com/45409515/77764323-dd22f780-7012-11ea-9d4f-0a5d9f6cfe6f.png">


Just a question @developpeurweb . Now that we have returns too, I tested this, and the focus is missing on the "Returns" tab. We can't navigate to that tab using keyboard. Do we have a ticket to tackle this?

<img width="1275" alt="Screen Shot 2020-03-27 at 10 06 09 AM" src="https://user-images.githubusercontent.com/45409515/77764516-2410ed00-7013-11ea-842e-52c1ae19510b.png">

**Comment by kacperknapik** [comment]
**QA results**
- focus works correctly now
- Firefox ✔️ 
- Safari ✔️ 
- Chrome ✔️
